broker/src/main/java/io/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -185,6 +190,20 @@ protected void onActorCloseRequested() {
     containers.forEach(ExporterContainer::close);
   }
 
+  @Override
+  protected void handleFailure(final Exception failure) {
+    LOG.error("Actor '{}' failed in phase {} with: {} .", name, actor.getLifecyclePhase(), failure);
+    actor.fail();
+
+    if (failure instanceof UnrecoverableException) {
+      healthStatus = HealthStatus.DEAD;
+    } else {
+      healthStatus = HealthStatus.UNHEALTHY;
```

Comment:
Was just wondering if this would break any current assumptions. Are there cases where we want to ignore unhandled exceptions and do not want to close the exporter and mark it as unhealthy?

Reply:
Not that I know of but we can be conservative and keep it the same for other exceptions

Reply:
Actually, it might also be misleading to registered listeners if failures are being implicitly ignored. AFAIK, there's no reason not to report them as unhealthy

Reply:
No need to remove this. I was just wondering if it is the expected semantics. It is good to mark it as unhealthy and close the broker if it actually breaks the exporting.